### ISSUE TITLE
Cannot open rom and lua script at once on Ubuntu

### ISSUE THREAD

**Issue by neauoire** [TASVideos/fceux#97]
## Working

```
fceux cart.nes
```

![Screenshot from 2020-01-30 18-31-16@2x](https://user-images.githubusercontent.com/1455726/73499713-d9665200-438e-11ea-8de6-73c3c0472e4f.png)

## Not Working

```
fceux cart.nes -lua debug.lua
```

![Screenshot from 2020-01-30 18-31-42@2x](https://user-images.githubusercontent.com/1455726/73499861-4679e780-438f-11ea-84ff-682180a5d44f.png)

It seems like when a lua script is present in the path, it fails to find both the `.nes` and ignores the `-lua` flag. Following this [docs](http://www.fceux.com/web/help/fceux.html?CommandLineOptions.html), running `fceux 2.2.2`

**Comment by mjbudd77** [comment]
I looked into this and it appears that the command line syntax in the linux SDL wrapper is not the same as its windows based version. The SDL command line seems to have been changed to support both unix style short and long option arguments. In your example, it failed to recognize -lua as an option and treated all three arguments as a ROM input file. Using the linux command line, the proper way to load both a ROM and lua file at startup would be:
fceux  ROM.nes   --loadlua  debug.lua
The command line options can also be seen by running:
fceux  --help

**Comment by neauoire** [comment]
Do I need to do something special to the path in that case?

```
derulin@villamoirai ~/R/s/donsol> cd tools/
derulin@villamoirai ~/R/s/d/tools> ls
asm6*  debug-zeropage.lua
derulin@villamoirai ~/R/s/d/tools> cd ..
derulin@villamoirai ~/R/s/donsol> fceux cart.nes --loadlua tools/debug-zeropage.lua
```

**Comment by mjbudd77** [comment]
I believe I have found the issue, the lua-engine script file load function is changing the current working directory (CWD) to the same directory in which script file exists.  But later in the function it attempts to use the relative path again to open the file which of course is going to fail since changing the CWD  invalidates the previously specified relative path. In your example, it fails since it is effectively doing a`cd tools; open tools/debug-zeropage.lua`
I have corrected this by forcing the code to resolve any relative paths to lua files from the command line to an absolute path before passing it to the lua-engine. I currently have the change grouped in with a merge request a number of big feature updates for the linux GUI. I do not know how long it will take for the fix to be accepted.
